go/citogo/main.go

```diff
@@ -67,6 +68,7 @@ func (r *runner) parseArgs() (err error) {
 	flag.BoolVar(&r.opts.NoCompile, "no-compile", false, "specify flag if you've pre-compiled the test")
 	flag.StringVar(&r.opts.TestBinary, "test-binary", "", "specify the test binary to run")
 	flag.StringVar(&r.opts.Timeout, "timeout", "60s", "timeout (in seconds) for any one individual test")
+	flag.IntVar(&r.opts.Pause, "pause", 0, "pause that many seconds between each test")
```

Comment:
Does an `IntVar` work with "2s" passed in?  Maybe it should be a `DurationVar`?

Reply:
mess up, thx